apps/condo/domains/user/oidc/adapter.js

```diff
@@ -0,0 +1,146 @@
+const Redis = require('ioredis')
+const isEmpty = require('lodash/isEmpty')
+
+const DEBUG = false
```

Comment:
May be move this to environment or remove completely?

Reply:
I think I'll remove it. in fact, you will have to add this when you are engaged in the debag of this task